### PR TITLE
add elm-hot-loader

### PR DESCRIPTION
**Pull request by fluxxu**
Thanks!

### CODE REVIEW

File: src/Main.elm [hunk 1]
```diff
@@ -54,4 +64,4 @@ styles =
       , ( "padding-bottom", "20px" )
       , ( "text-align", "center" )
       ]
-  }
+  }
```
Comment (gothy) [inline src/Main.elm]: newline would be great :)

**Comment by gothy** [comment]
:+1:

File: src/Main.elm (hunk 1 above)
Comment (fluxxu) [inline src/Main.elm]: Done.

**Comment by moarwick** [comment]
@fluxxu: thanks!  Seems to work, except `npm run build` throws an error from Uglify: `Unexpected token punc «,», expected punc «:» [./~/elm-hot-loader!./~/elm-webpack-loader!./src/Main.elm:10941,0]`  Appears to be in `findModulePath()`?  Any ideas?  I can still keep the merge, will just comment out the minification step for now..

**Comment by moarwick** [comment]
@fluxxu: nvm, issue fixed. i had to change the webpack config so your hot-loader is applied in development only.

**Comment by fluxxu** [comment]
Glad it's working  :)

**Pull request merged**